File: CalculatorMemory.cs
namespace Calculator
{
    public class CalculatorMemory
    {
        private double _memory;

        public void Add(double value) => _memory += value;

        public void Subtract(double value) => _memory -= value;

        public double Recall() => _memory;

        public void Clear() => _memory = 0;
    }
}

File: src/main/java/com/calculator/CalculatorMemory.java
package com.calculator;

/**
 * Manages the calculator memory register.
 */
public class CalculatorMemory {

    private double memory;

    public void add(double value) {
        memory += value;
    }

    public void subtract(double value) {
        memory -= value;
    }

    public double recall() {
        return memory;
    }

    public void clear() {
        memory = 0;
    }
}
